web-bundle/src/main/java/com/graphhopper/replica/StreetEdgeExporter.java

```diff
@@ -170,6 +206,13 @@ public static void writeStreetEdgesCsv(GraphHopper configuredGraphHopper,
         }
     }
 
+    private static int getSpeedcms(EdgeIteratorState edgeIterator, DecimalEncodedValue avgSpeedEnc) {
```

Comment:
I factored this out sort-of-by-accident along the way to the structure that's present here, but I'm not mad about it. Still, it's used once now, and could easily be inlined.